cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/DistributedProgramRuntimeService.java

```diff
@@ -200,7 +200,7 @@ public synchronized RuntimeInfo lookup(final RunId runId) {
           continue;
         }
 
-        Id.Program programId = Id.Program.from(matcher.group(2), matcher.group(3), matcher.group(4));
+        Id.Program programId = Id.Program.from(matcher.group(2), matcher.group(3), appType, matcher.group(4));
```

Comment:
Not sure why the local variable was previously named `appType`. Anyway, the local variable is not needed, so I can remove it.